app/views/datasets/_meta_data.html.erb

```diff
@@ -36,9 +36,11 @@
             <span itemprop="name"><%= dataset_location(@dataset) %></span>
           </dd>
 
-          <% if @dataset.topic %>
-            <dt><%= t('.topic') %>:</dt>
-            <dd><%= @dataset.topic['title'] %>
+          <dt><%= t('.topic') %>:</dt>
+          <% if(@dataset.topic) %>
+            <dd><%= @dataset.topic['title'] %></dd>
```

Comment:
Very minor, but we're duplicating the `<dd>` tags. You could be bit fancy:

```erb
<dd><%= @dataset&.dig(:title) || t('.topic_not_added') %></dd>
```

Reply:
Can't use dig on ActiveRecord instances :-(